mmdet/datasets/builder.py

```diff
@@ -11,6 +13,24 @@ def _concat_dataset(cfg, default_args=None):
     seg_prefixes = cfg.get('seg_prefix', None)
     proposal_files = cfg.get('proposal_file', None)
 
+    if cfg.get('img_prefix_auto', False):
+        del cfg['img_prefix_auto']
+        assert img_prefixes is None
+        if cfg['type'] == 'CustomCocoDataset':
+            # assuming following dataset structure:
+            # dataset_root
+            # ├── annotations
+            # │   ├── instances_train.json
+            # │   ├── ...
+            # ├── images
+            #     ├── image_name1
+            #     ├── image_name2
+            #     ├── ...
+            # and file_name inside instances_train.json is relative to dataset root
+            img_prefixes = [os.path.join(os.path.dirname(ann_file), '..') for ann_file in ann_files]
```

Comment:
Shouldn't the path contain `images` as well?

Reply:
this is a matter of some agreement. Added